datastore/load.go

```diff
@@ -369,7 +369,7 @@ func setVal(v reflect.Value, p Property) (s string) {
 			if ok {
 				s := micros / 1e6
 				ns := micros % 1e6
-				v.Set(reflect.ValueOf(time.Unix(s, ns)))
+				v.Set(reflect.ValueOf(time.Unix(s, ns).In(time.UTC)))
```

Comment:
Mind adding a test that ensures the times are returned in UTC?

Reply:
+1, might be easier said than done though :)

Reply:
test added.